src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -371,6 +371,10 @@
     <value>Open the tab with tabTitle overriding default title and suppressing title change messages from the application</value>
     <comment>{Locked="\"tabTitle\""}</comment>
   </data>
+  <data name="CmdInheritEnvDesc" xml:space="preserve">
+    <value>Open the new tab by inheriting the environment variables, rather than reloading a fresh environment block. This defaults to set when a "command" is passed. </value>

```

Comment:
nit: not necessarily a tab (!) and it is not necessarily done "by" inheriting (phrasing is odd)